rfcs/050-concepts-api/README.md

```diff
@@ -0,0 +1,194 @@
+
+  // not locations
+  "urls": [
+    "https://en.wikipedia.org/wiki/Florence_Nightingale"
+  ],
+
+  // cf productionEvent?
+  "dates": [ { date, meaning } ]
+  "places": [ { place, meaning } ]
+  "birthDate/place"
+  "deathDate/place"
+
+  "thumbnail": { … },
+
+  "relatedConcepts": [
+    {
+      "id": "asoiham1",
+      "label": "Crimea",
+      "type": "Place"
+    }
+  ]
+}
+```
+
+Notes:
+
+-   The `id` and `identifiers` fields are the same as on existing works/images.
+
+-   The `label` and `alternativeLabels` fields are named analogously to `title` and `alternativeTitles` on works.
+
+-   The `type` will be an enumeration that includes Person, Organisation, Meeting, Subject.
+    For the initial implementation we'll likely have a few subtypes of Person but not of Subject.
+
+    The alternative is to have `type: Concept` and do subtypes through something similar to workType/format, but I'm not sure that adds anything.
+
+    This is a value users would be able to filter on (if we provide an endpoint for listing/searching concepts).
+
+-   The `description` should provide enough context to help users distinguish subjects/people with the same label but different meaning.
+    This will likely be drawn from LCSH or Wikidata initially, but at some point we might write our own content here.
+
+    Q: Is `description` the best name for this field, or should it be called something different?
+
+-   The `urls` list points to this subject/person elsewhere.
+    We aren't trying to replicate LCSH/MESH/Wikidata; if somebody wants to read more in-depth about a given concept, we should link them to an external entry.
+
+    In works we use digital locations in `work.items.locations` to link to external resources.
+    We don't want to reuse that structure, which is more complicated than we need here.
+
+    Q: Aside from the URL, what other values do we want for these links?
+    e.g. link label
+
+    Q: Do we need these values at all?
+    If we put the Wikidata ID in the list of identifiers, do we leave clients to construct a URL, or do we provide it in the API?
+
+-   The `date`/`place` information is somewhat speculative.
+
+    We think we might include birth/death information on per-person pages; we could also link to significant dates and places and use that as a way to link between concepts.
+    This is similar to production events on works.
+
+    We can start with a `birthDate/birthPlace/deathDate/deathPlace` field on the initial version of the API, and decide if we want to put this information on per-person pages.
+    If yes, we can think about how to model this properly.
+    If no, we can scrap all of these fields.
+
+-   The `relatedConcepts` field lets us link between concept pages.
+    It contains a simplified subset of a Concept's fields; enough to render a link but nothing more.
+
+    We don't want to describe how two concepts are related; this is significantly more complicated.
+
+    Q: Is relatedness symmetric?
```

Comment:
Good question, I would question the naming `related`

Reply:
I've changed it to `connected` for now; I don't think that's the final term, but it's enough for the discussion I want to have here.